toolkit/tools/pkg/imagemodifierlib/imagemodifier.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagemodifierlib
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/imagecustomizerapi"
+)
+
+func ModifyImageWithConfigFile(configFile string) error {
+	var err error
+
+	var config imagecustomizerapi.Config
+	err = imagecustomizerapi.UnmarshalYamlFile(configFile, &config.SystemConfig)
```

Comment:
@cwize1  Here I pass &config.SystemConfig as opposed to &config,  so from customer side they don't need to prefix `SystemConfig:` at the top of the yaml. (e.g. Trident doesnt have this SystemConfig instead they have osconfig:, the entities under osconfig will be written to a yaml). let me know if this makes sense or not.

Reply:
Yeah, that fine. Though, I would use `var systemConfig imagecustomizerapi.SystemConfig`. No point using the `Config` type if you are not using it.